### PR TITLE
GWMS documents reviewed

### PR DESCRIPTION
**Pull request by mambelli**
GWMS documents reviewed

Most changes are directly in the documents (fixed some links and some text). Feel free to accept the changes or modify them.
Then in the Factory documentation the local_submit.sh example in Factory/Validating GlideinWMS Factory is missing
And there is some OSG document that is referenced in the twiki (camelcase names) but here the links are not done automatically.

Thanks,
Marco

### CODE REVIEW

File: docs/other/install-gwms-factory.md [hunk 1]
```diff
@@ -195,35 +196,34 @@ For the security:
...
+An example test entry is provided in the default GlideinWMS configuration file.  At the very least, you will need to modify the entry line:
 
       :::xml
-      %RED%# These lines are from the configuration of v 3.x%ENDCOLOR%
        <entry name="%RED%ENTRY_NAME%ENDCOLOR%" enabled="True" auth_method="grid_proxy" trust_domain="OSG" 
        gatekeeper="%RED%gatekeeper.domain.tld/jobmanager-type%ENDCOLOR%" gridtype="gt2" rsl="(queue=default)(jobtype=single)"
        schedd_name="%RED%schedd_glideins2@FACTORY_HOSTNAME%ENDCOLOR%" verbosity="std" work_dir="OSG">
 
-You will need to modify the entry **name** and **gatekeeper**.  This will determine the gatekeeper that you access.  Specific gatekeepers often require specific "rsl" attributes that determine the job queue that you are in or other attributes.  Add them in the _rsl_ attribute.  
+You will need to modify the entry **name** and **gatekeeper**.  This will determine the gatekeeper that you access.  Specific gatekeepers often require specific "rsl" attributes that determine the job queue that you are ubmitting to, or other attributes.  Add them in the _rsl_ attribute.
```
Comment (llobato) [inline docs/other/install-gwms-factory.md]: Need to change " ubmitting to" by "submitting to"

File: docs/other/install-gwms-frontend.md [hunk 2]
```diff
@@ -840,7 +835,7 @@ If the startup script of the frontend is failing, check the log file for errors
 
 If you find errors like *"Exception occurred: ... 'ExpatError: no element found: line 1, column 0\\n'\]"* and *"IOError: \[Errno 9\] Bad file descriptor"* you may have an empty status file (`/var/lib/gwms-frontend/vofrontend/monitor/group_*/frontend_status.xml`) that causes GlideinWMS Frontend not to start. The glideinFrontend crashes after a XML parsing exception visible in the log file ("Exception occurred: ... 'ExpatError: no element found: line 1, column 0\\n'\]").
 
-Remove the status file. Then start the frontend. The fronten will be fixed in future versions to handle this automatically.
+Remove the status file. Then start the frontend. The Fronten will be fixed in future versions to handle this automatically.
```
Comment (llobato) [inline docs/other/install-gwms-frontend.md]: Should be "FrontEnd" instead of "Fronten"

**Review by llobato — APPROVED** [review]

**Review by llobato — APPROVED** [review]

File: docs/other/install-gwms-frontend.md (hunk 2 above)
Comment (mambelli) [reply docs/other/install-gwms-frontend.md]: Frontend (for consistency only the first letter is capitalized)

**Comment by mambelli** [comment]
I added your suggested changes. Only you can merge to your branch.

**Pull request merged**